diff --git a/colors/solarized.vim b/colors/solarized.vim
index 70f5223..9b3078e 100644
--- a/colors/solarized.vim
+++ b/colors/solarized.vim
@@ -135,25 +135,30 @@
 " environment.
 
 " Terminals that support italics
-let s:terms_italic=[
-            \"rxvt",
-            \"gnome-terminal"
-            \]
-" For reference only, terminals are known to be incomptible.
-" Terminals that are in neither list need to be tested.
-let s:terms_noitalic=[
-            \"iTerm.app",
-            \"Apple_Terminal"
-            \]
-if has("gui_running")
-    let s:terminal_italic=1 " TODO: could refactor to not require this at all
+"let s:terms_italic=[
+"            \"rxvt",
+"            \"gnome-terminal"
+"            \]
+"" For reference only, terminals are known to be incomptible.
+"" Terminals that are in neither list need to be tested.
+"let s:terms_noitalic=[
+"            \"iTerm.app",
+"            \"Apple_Terminal"
+"            \]
+"if has("gui_running")
+"    let s:terminal_italic=1 " TODO: could refactor to not require this at all
+"else
+"    let s:terminal_italic=0 " terminals will be guilty until proven compatible
+"    for term in s:terms_italic
+"        if $TERM_PROGRAM =~ term
+"            let s:terminal_italic=1
+"        endif
+"    endfor
+"endif
+if has("gui_running") || ( has("unix") && system("tput sitm") == "\033[3m" )
+    let s:terminal_italic=1
 else
-    let s:terminal_italic=0 " terminals will be guilty until proven compatible
-    for term in s:terms_italic
-        if $TERM_PROGRAM =~ term
-            let s:terminal_italic=1
-        endif
-    endfor
+    let s:terminal_italic=0
 endif
 
 " }}}
